front/src/applications/stdcm/hooks/useStdcm.ts

```diff
@@ -68,7 +68,10 @@ const useStdcm = ({
       { skip: !osrdconf.rollingStockID }
     );
 
-  const { speedLimitByTag } = useStoreDataForSpeedLimitByTagSelector({ isStdcm: true });
+  useStoreDataForSpeedLimitByTagSelector({
```

Comment:
I have to confess I'm not fully convinced of what this hook call does now.

Reply:
Hum me too. It's only used to dispatch the default speed limit by tag. Maybe we can remove it in another commit/PR ?